tests/testthat/test-forecast.R

```diff
@@ -157,6 +157,35 @@ test_that("output of as_forecasts() is accepted as input to score()", {
   expect_equal(score_check, suppressMessages(score(as_forecast_binary(example_binary))))
 })
 
+
+# as_forecast.forecast_nominal() -----------------------------------------------
+test_that("as_forecast.forecast_nominal() works as expected", {
```

Comment:
I think this test is checking that all forecasts have all necessary rows, even if predicted probabilities is zero. maybe that should be a more specific description of the test here 
```
test_that("as_forecast.forecast_nominal() breaks when rows with zero probability are missing")
```

Reply:
Note for the future that you can use the suggestion interface (the piece of paper in the comment bar with +- on it) to make this a code suggestion that can be batched merged.